frontend/src/metabase/public/components/widgets/EmbedModalContent.unit.spec.tsx

```diff
@@ -25,7 +25,7 @@ describe("EmbedModalContent", () => {
 
   it("should render parameters", () => {
     const parameters = [
-      { name: "My param", slug: "my_param", type: "category" },
+      { name: "My param", slug: "my_param", type: "string/=" },
```

Comment:
This doesn't make any difference to the test result, but I just couldn't find a way to set the parameter type to `category` at the moment.

Looking at `PARAMETER_OPERATOR_TYPES` you can see that there's no way to get a parameter with `category` type, unless I mistook this https://github.com/metabase/metabase/blob/6476e38ac0b33fcc3a1ac069fd97b01347414441/frontend/src/metabase-lib/parameters/constants.ts#L3

Reply:
That makes sense

Reply:
Could we omit this change? I think it's important to test `category` type. Maybe the root cause is that `Parameter` in `metabase-types/api` has a wrong type for `type` field. It should be `string` like it is in `metabase-types/types`. If that's the case here, let's fix the type in `api`

Reply:
I can revert this change, at first I thought `category` was impossible, but you have stated it is possible.